ui/admin/app/templates/onboarding/success.hbs

```diff
@@ -0,0 +1,94 @@
+{{page-title (t 'titles.success')}}
+<Rose::Layout::Centered class='create-resources'>
+  <Rose::Frame>
+    <:header>
+      <h2>{{t 'onboarding.success.titles.connect'}}</h2>
+    </:header>
+    <:body>
+      {{! We have created }}
+      <div class='create-resources-item'>
+        <span class='create-resources-item-header'>
+          <FlightIcon
+            @name='check-circle'
+            @color='var(--token-color-foreground-success)'
```

Comment:
this should live in scss, we could use the class`create-resources-item-header` above to target the icon

Reply:
@DhariniJeeva 
Not that I have a strong opinion, but what is the reason this should live in scss? I think this is the advantatge of tokens and the <FlightIcon> component spect [here link](https://hds-website-hashicorp.vercel.app/icons/usage-guidelines?tab=code) shows this as a way of using it.

Reply:
Separation of concerns, the beauty and readability of this expression notwithstanding.  I agree that we should not mix styles and templates in this manner, even though there is an example of this usage.  I'm actually surprised icons don't use `currentColor` so that we can thus use [HDS color classes](https://hds-components-hashicorp.vercel.app/foundations/colors) or overrides in our stylesheets.

Reply:
@DhariniJeeva @randallmorey 
I understand separation of concerns, but this is a pattern we (almost all devs in Boundary) are already following, just look on your code editor and search for `@color` in our codebase. From those 22 matches follow this pattern, using `@color` in the template.
HDS give us the `@color` attribute and tokens for a reason. 

Since this looks to me as an opinionated "rule". I will suggest to bring it to tech time for discussion and whatever decission we take about it, we should documented. Because opinionated patterns should be documented, otherwise... will not be consistency with it.

Reply:
@randallmorey
Regarding the `currentColor` comment. That was my first approach but HDS did not support it, I opened [an slack thread](https://hashicorp.slack.com/archives/C7KTUHNUS/p1674860484678539) that mentions such thing.

Reply:
The existing usage of `@color` is specifically with semantic color names, not style tokens.  Using style tokens is specifically the usage I am commenting on here.

Reply:
@randallmorey 
I understand you were commenting on the usage of a token. And as per my understanding is opinionated.
As per understanding of HDS seems like is opinionated too, this is extracted from [here](https://hds-components-hashicorp.vercel.app/foundations/colors):

"There are two different ways to apply a color to a UI element, via design tokens or via CSS helper classes.

Which one to use will depend on the context: since both refer to the same design token values, from the design system perspective there's no difference between the two methods (it's more of a preference for the "consumer" codebase)."

If you don't mind, I would like to bring this to FE tech time, discuss it and document it somewhere, so we are clear on what patterns to follow.